src/cf-tables.less

```diff
@@ -370,11 +380,24 @@
 }
 
 .table__sortable {
+    button.sortable {
+        width: 100%;
+        height: 100%;
+        padding: inherit;
+        margin: inherit;
+        border: none;
+        background: none;
+        font-family: inherit;
+        font-weight: inherit;
+        outline: none;
+        text-align: left;
+    }
```

Comment:
Yes! Except cf-tables doesn't currently require cf-buttons. Should we file this as an Issue that we fix once cf-tables makes it into the Voltrazord? Or is there a better fix?